pkg/ddevapp/task.go

```diff
@@ -164,17 +164,7 @@ func NewTask(app *DdevApp, ytask YAMLTask) Task {
 			}
 			return t
 		}
-		util.Warning("Invalid exec_raw value, not executing it: %v", e)
-
-	} else if e, ok = ytask["exec"]; ok {
-		if v, ok := e.(string); ok {
-			t := ExecTask{app: app, exec: v}
-			if t.service, ok = ytask["service"].(string); !ok {
-				t.service = nodeps.WebContainer
-			}
-			return t
-		}
-		util.Warning("Invalid exec value, not executing it: %v", e)
```

Comment:
The above else-if condition at the line 145 is exactly the same, I believe this code is unreachable.